internal/provider/log_stream_resource.go

```diff
@@ -183,6 +183,37 @@ func (r *logStreamResource) Schema(_ context.Context, _ resource.SchemaRequest,
 					},
 				},
 			},
+			"crowdstrike_http_event_collector": schema.SingleNestedAttribute{
+				Description: "Log Stream configuration for the Crowdstrike HTTP Event Collector (HEC) destination type.",
+				Optional:    true,
+				Attributes: map[string]schema.Attribute{
+					"hec_host_port": schema.StringAttribute{
```

Comment:
```suggestion
					"host_port": schema.StringAttribute{
```
Since this is already in a block that references HEC, the prefix is unnecessary.